utils/utils.go

```diff
@@ -87,20 +90,36 @@ func HexDecodeInto(dst []byte, h []byte) error {
 	return nil
 }
 
-// CheckBigIntInField checks if given big.Int fits in a Field Q element
-func CheckBigIntInField(a *big.Int, q *big.Int) bool {
-	if a.Cmp(q) != -1 {
+// CheckBigIntInField checks if given *big.Int fits in a Field Q element
+func CheckBigIntInField(a *big.Int) bool {
+	if a.Cmp(constants.Q) != -1 {
 		return false
 	}
 	return true
 }
 
-// CheckBigIntArrayInField checks if given big.Int fits in a Field Q element
-func CheckBigIntArrayInField(arr []*big.Int, q *big.Int) bool {
+// CheckBigIntArrayInField checks if given *big.Int fits in a Field Q element
+func CheckBigIntArrayInField(arr []*big.Int) bool {
 	for _, a := range arr {
-		if !CheckBigIntInField(a, q) {
+		if !CheckBigIntInField(a) {
 			return false
 		}
 	}
 	return true
 }
+
+// CheckElementArrayInField checks if given *ff.Element fits in a Field Q element
+func CheckElementArrayInField(arr []*ff.Element) bool {
+	for _, aE := range arr {
+		a := big.NewInt(0)
+		aE.ToBigIntRegular(a)
+		if !CheckBigIntInField(a) {
+			return false
+		}
+	}
+	return true
+}
+
+func NewElement() *ff.Element {
```

Comment:
I would suggest either:
1. Remove this function and just do `&ff.Element{}` instead, which does the same
2. Move the `NewElement` function to `ff/` in a new file.

Reply:
At the beginning `NewElement` was outside the `ff` package, as is the package generated by `goff` and didn't wanted to add non generated code there, but now I've moved this function into `ff/util.go`.